conf/modules/annotate_structural_variants.config

```diff
@@ -37,14 +37,14 @@ process {
 
     withName: '.*ANNOTATE_STRUCTURAL_VARIANTS:ENSEMBLVEP_SV' {
         ext.args = { [
-            '--dir_cache vep_cache',
-            '--dir_plugins vep_cache/Plugins',
+            '--dir_cache cache',
+            '--dir_plugins cache/Plugins',
             '--plugin pLI,pLI_values.txt',
             '--appris --biotype --buffer_size 100 --canonical --cache --ccds',
             '--compress_output bgzip --distance 5000 --domains',
             '--exclude_predicted --force_overwrite --format vcf',
-            '--fork 4 --hgvs --humdiv --max_sv_size 248956422 --merged',
-            '--no_progress --numbers --per_gene --polyphen p',
+            '--hgvs --humdiv --max_sv_size 248956422 --merged',
+            '--dont_skip --no_progress --numbers --per_gene --polyphen p',
```

Comment:
Did adding this option help?

Reply:
Sorry! I was testing different options, and this one slipped in. Good catch! I removed it.